packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,279 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as KeyCode from 'keycode-js';
+import '../_elementPolyfill';
+import WorklistDataGridUtils from '../utils/WorklistDataGridUtils';
+
+const propTypes = {
+  /**
+   * String identifier of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+
+  coordinates: PropTypes.shape({
+    x: PropTypes.number,
+    y: PropTypes.number,
+  }),
+
+  /**
+   * Boolean value to indicate if the cell can accept focus.
+   */
+  acceptsFocus: PropTypes.bool,
+
+  /**
+   * Boolean indicating whether the Cell is actively selected.
+   */
+  isSelected: PropTypes.bool,
+
+  /**
+   * @private
+   * Boolean indicating whether the cell is a row selection cell.
+   */
+  isRowSelectionCell: PropTypes.bool,
+
+  /**
+   * String that labels the cell for accessibility.
+   */
+  ariaLabel: PropTypes.string,
+
+  /**
+   * Class name for the cell.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Function that is called when a selectable cell is selected. Parameters: `onCellSelect(rowId, columnId)`
+   */
+  onCellSelect: PropTypes.func,
+
+  onCellSelectionChange: PropTypes.func,
+  onMoveCellFocus: PropTypes.func,
+  onNavigationModeChange: PropTypes.func,
```

Comment:
Looking at your design and our conversation today, I believe this should be completely encapsulated inside the Cell component.  The WorklistDataGrid object is just passing a useState hook.  In that case, it is causing the entire WorklistDataGrid to be re-rendered. However, I don't think that is the expected behavior here.